README.md

```diff
@@ -348,6 +348,14 @@ On a `Webhook` instance you may also call:
 $webhook->delete();
 ```
 
+### Site Commands
+
+```php
+$forge->createSiteCommand($serverId, $siteId, array $data);
```

Comment:
I would call this `executeSiteCommand`.

Reply:
Changed.  I think I had written this up prior to the change from create to execute, so I had the old naming.